src/main.rs

```diff
@@ -56,8 +81,9 @@ fn main() {
         ("git", Some(_matches)) => {
             println!("hermit git is not yet implemented");
         }
-        ("init", Some(_matches)) => {
-            println!("hermit init is not yet implemented");
+        ("init", Some(matches)) => {
+            let shell_name = matches.value_of("").unwrap_or("default");
```

Comment:
If `init` has arguments available to it we can actually match on the subcommand arg and then create both the default shell and other optionally named shells.